scipy/optimize/tests/test_linprog.py

```diff
@@ -359,6 +357,86 @@ def f(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None, bounds=None):
 
         assert_raises(ValueError, f, [1, 2], A_ub=np.zeros((1, 1, 3)), b_eq=1)
 
+    def test_bounds_fixed(self):
```

Comment:
I'd like to see a more involved test that has constraints, because none of these actually run the main linprog methods due to:
```
    # no constraints indicates that problem is trivial
    if A_eq.size == 0 and A_ub.size == 0:
         # code that solves trivial problems
```

Reply:
No problem.